src/main/java/org/opensearch/knn/index/engine/faiss/FaissHNSWPQEncoder.java

```diff
@@ -69,4 +72,13 @@ public class FaissHNSWPQEncoder implements Encoder {
     public MethodComponent getMethodComponent() {
         return METHOD_COMPONENT;
     }
+
+    @Override
+    public CompressionLevel calculateCompressionLevel(
+        MethodComponentContext methodComponentContext,
+        KNNMethodConfigContext knnMethodConfigContext
+    ) {
+        // TODO: For now, not supported out of the box
+        return CompressionLevel.NOT_CONFIGURED;
```

Comment:
why not 1x here?

Reply:
I rather use NOT_CONFIGURED because 1x is not correct in that it wont be 1x.